tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -234,6 +234,10 @@
         border-radius: 10px;
       }
 
+      .inference-settings-disabled {
+         display: none;
```

Comment:
curious, since we close the dialog when local=false, do we absolutely require this?

Reply:
removed